### PR TITLE
Upload API of all files from client to server

### PR DESCRIPTION
**Pull request by youralien**
It creates a project directory with all neccessary files on the cloud

Summary:
- needed to rename `utils` to `traffic_cloud_utils` since there was name conflicts with TrafficIntelligence.utils
- renamed `cloud` to `trafficcloud`.  Must add this directory to your system path.  i.e. in your .bashrc, `export PYTHONPATH="${PYTHONPATH}:/home/user/Documents/TrafficCloud/trafficcloud"
- UploadHandler takes in file upload binaries, passes it to the api object which creates files in the appropriate project directory structure, and returns back a UUID for the project which is sent back to the client.

### CODE REVIEW

File: app/handlers/cloud/pm.py [hunk 1]
```diff
@@ -32,58 +33,64 @@ def __init__(self,files):
...
+                newcfg.write(nline1+nline2+old)
 
             svms = ["modelBV.xml", "modelPB.xml", "modelPBV.xml", "modelPV.xml"]
             for svm in svms:
-                copy("default/{}".format(svm), os.path.join(pr_path, svm))
+                copy(os.path.join(os.path.dirname(os.path.realpath(__file__)), "default/{}".format(svm)), os.path.join(pr_path, svm))
 
-            video_dest = os.path.join(pr_path, os.path.basename(self.videopath))
-            copy(self.videopath, video_dest)
+            # video_dest = os.path.join(pr_path, os.path.basename(self.videopath))
```
Comment (segerphilip) [inline app/handlers/cloud/pm.py]: Probably remove this, don't need commented out code because vcs.

File: app/handlers/cloud/pm.py [hunk 2]
```diff
@@ -181,7 +188,7 @@ def load_project(folder_path): # main_window):
 #     else:
 #         print ("{} does not exist. No points loaded.".format(corr_path))
 
-def load_results(main_window):
-    if os.path.exists(os.path.join(ac.CURRENT_PROJECT_PATH, "homography", "homography.txt")):
-        if os.path.exists(os.path.join(ac.CURRENT_PROJECT_PATH, "run", "results.sqlite")):
-            plot_results(main_window)
+# def load_results(main_window):
```
Comment (segerphilip) [inline app/handlers/cloud/pm.py]: Same, remove commented code.

File: app/handlers/upload.py [hunk 3]
```diff
@@ -3,21 +3,49 @@
...
+
+        name_body_dict = self.parse_request(self.request.files)
+        uuid = api.saveFiles(name_body_dict)
+
+        # TODO(rlouie): This is the next step!!!
+        # api.runTrajectoryAnalysis(uuid)
+
+        self.finish("Everything is uploaded!! Check uploads folder")
+
+        # for k, v in self.request.files.iteritems():
```
Comment (segerphilip) [inline app/handlers/upload.py]: Remove

**Review by segerphilip — COMMENTED** [review]
Not sure what the .swp file is doing here, probably don't want that. Also, remove the commented out code. Test then 🚢

**Comment by youralien** [comment]
in order to make this change work, you have to add this to your .bashrc

`export PYTHONPATH="${PYTHONPATH}:/home/user/Documents/TrafficCloud/"`

**Pull request merged**